openshift-examples/keycloak.yaml

```diff
@@ -4,39 +4,39 @@ metadata:
   name: keycloak
   namespace: openshift-config
```

Comment:
@andreaTP Maybe OT / unrelated to fixing the original issue, but does the namespace field needs to be present there?

Reply:
removed 👍